storage.go

```diff
@@ -157,6 +157,25 @@ func (s *Storage) createDir(ctx context.Context, path string, opt pairStorageCre
 	return
 }
 
+func (s *Storage) createLink(ctx context.Context, path string, target string, opt pairStorageCreateLink) (o *Object, err error) {
+	rt := s.getAbsPath(target)
+	rp := s.getAbsPath(path)
+
+	o = s.newObject(true)
+	o.ID = rp
+	o.Path = path
+	o.SetLinkTarget("/" + rt)
```

Comment:
Why add `/` here?

Reply:
Because when I tested it, I found that each time the target was missing a `/` compared to the expect target.

Reply:
> Because when I tested it, I found that each time the target was missing a `/` compared to the expect target.

It's not the root cause.

Object storage services like oss, s3 will not have absolute paths like fs. So we need to convert between internal object key and paths that exposed to users.